### PR TITLE
Gives the CMO a laser pointer to play with their cat

### PR DESCRIPTION
**Pull request by LarissaMayrink**
## About The Pull Request
Adds a laser pointer to the cmo starting kit on the right hand. 

## Why It's Good For The Game
CMO has a cat but no laser pointer to play with runtime. Playing with runtime is good.

## Changelog

:cl:
add: Powercreeps the CMO by giving them a laser pointer
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/chief_medical_officer.dm [hunk 1]
```diff
@@ -68,6 +68,7 @@
 	shoes = /obj/item/clothing/shoes/sneakers/blue
 	l_pocket = /obj/item/pinpointer/crew
 	l_hand = /obj/item/storage/medkit/surgery
+	r_hand = /obj/item/laser_pointer
```
Comment (Fikou) [inline code/modules/jobs/job_types/chief_medical_officer.dm:71]: put it in their pocket instead

Comment (LarissaMayrink) [reply code/modules/jobs/job_types/chief_medical_officer.dm:71]: ok, one sec

Comment (Fikou) [reply code/modules/jobs/job_types/chief_medical_officer.dm:71]: (this will save you from editing the unit tests and its a free slot so why put it in hand

Comment (GuillaumePrata) [reply code/modules/jobs/job_types/chief_medical_officer.dm:71]: `r_hand` is where the Plasmamen internals go when they spawn so it should be avoided for a good reason too.

Comment (LarissaMayrink) [reply code/modules/jobs/job_types/chief_medical_officer.dm]: done, changed

**Comment by Mothblocks** [comment]
Can we make this something in the maps instead? Seems like this is just going to get thrown away every round

**Comment by Iamgoofball** [comment]
only idiots throw away the laser pointer, it's a really good tool

**Pull request merged**